assets/js/blocks/cart-checkout/cart/full-cart/cart-line-item-row.js

```diff
@@ -8,8 +8,16 @@ import QuantitySelector from '@woocommerce/base-components/quantity-selector';
 import FormattedMonetaryAmount from '@woocommerce/base-components/formatted-monetary-amount';
 import { getCurrency } from '@woocommerce/base-utils';
 
+const ProductVariationDetails = ( { variation } ) => {
+	const variationsText = variation
+		.map( ( v ) => `${ v.attribute }: ${ v.value }` )
+		.join( ' / ' );
+
+	return <div>{ variationsText }</div>;
```

Comment:
Do you think it might be worth adding a CSS class to this `<div>` even if we don't use it? That would allow theme authors to style it if they want to.

Reply:
Good idea - I could imagine some stores might hide or emphasise different elements of the cart.  Added here: 443057e9ad73bee23af24b4a8595360834df7c58